eng/versioning/scan_for_unreleased_dependencies.ps1

```diff
@@ -34,7 +34,7 @@ Get-ChildItem -Path $serviceDirectory -Filter pom*.xml -Recurse -File | ForEach-
         $script:FoundPomFile = $true
         Write-Host "Found pom file with matching groupId($($inputGroupId))/artifactId($($inputArtifactId)), pomFile=$($pomFile)"
         $version = $xmlPomFile.project.version
-        if ($version -like '*-beta.*')
+        if ($version -like '*-beta.*' -or $version -like '*-beta')
```

Comment:
While this works you could also change it to `-match '.*-beta(\.\d*)?`

Reply:
> Agree with @weshaggard's sentiment here about thinking about whether we really want to support this. I think we should snap to our guidance around versioning. I can't think of a valid reason why we aren't keeping his consistent with the rest of the SDK.

I would agree if this were all of track 1 but it isn't, it's a couple of libraries. Other libraries like eventhubs and keyvault are following the version convention. I do think it's worth keeping the auto-increment and the changes were relatively save and small. It's either that or I start adding conditions based on track in the release yml based upon tracks and I'm reluctant to do so unless absolutely necessary.

Reply:
Good point. I'll make it locally and push another commit.